test/js-native-api/test_string/test_string.c

```diff
@@ -310,6 +310,58 @@ static napi_value TestPropertyKeyUtf16AutoLength(napi_env env,
                          auto_length);
 }
 
+static napi_value TestSetNamedPropertyLen(napi_env env, napi_callback_info info) {
```

Comment:
These new test functions are not called. Read more about named property tests at https://github.com/nodejs/node/blob/main/test/js-native-api/test_object/test_object.c#L250 and https://github.com/nodejs/node/blob/main/test/js-native-api/test_object/test.js#L118.